airflow-core/src/airflow/ui/src/i18n/config.ts

```diff
@@ -37,13 +37,16 @@ export const supportedLanguages = [
 export const defaultLanguage = "en";
 export const namespaces = ["common", "dashboard", "dags", "admin", "browse", "assets", "components"] as const;
 
+const baseUrl = document.querySelector("base")?.href ?? "http://localhost:8080/";
```

Comment:
Nit: Maybe that? I see we have places with that and places with `http://locahost:8080/` like here. Maybe keeping it relative to the current url is the best approach, instead of hardcoding `http://localhost:8080` in case the base url is not defined ?
```suggestion
const baseUrl = document.querySelector("base")?.href ?? "";
```

Reply:
Now that you mention it, we probably shouldn't have a default like that anywhere. If we want/need one, it should be [here](https://github.com/apache/airflow/blob/b20d6fe0ee183bc9d5498afb264cfb746d2dc070/airflow-core/src/airflow/config_templates/config.yml#L1370).

Reply:
Yes, and we don't want a default there either, because we want to default to the current 'location' of the front-end. (where it's served)